LICENSE.md

```diff
@@ -1,4 +1,8 @@
-Each sub-project folder has its own license terms.
+Each project subdirectory has its own license terms.
 
-* gnovm, gno.land - copyleft derived from AGPL3.0
-* tm2 - Apache2.0
+AGPL 3.0 (copyleft)
+- [Gno.land](https://github.com/gnolang/gno/blob/master/gno.land/LICENSE.md)
+- [GnoVM](https://github.com/gnolang/gno/blob/master/gnovm/LICENSE.md)
```

Comment:
I'm open to the idea, but I don't think the previous version was that bad. Does anyone have additional feedback?

By the way, when using internal links, I recommend using relative paths instead of URLs.

Reply:
<img width="1129" alt="Screenshot 2023-10-11 at 5 30 38 PM" src="https://github.com/gnolang/gno/assets/17755587/234e5ddb-ae59-4c49-9bd7-a29aad9ab86d">

We can do whatever you would prefer, but it felt like this was a request to clean up the presentation (esp. for non-technicals)